seyik/routes/index.js

```diff
@@ -0,0 +1,13 @@
+//routes/index.js
+
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+const postRouter = require("./postRouter");
+const likeRouter = require("./likeRouter");
+router.use("/users", userRouter.router);
```

Comment:
가독성을 위해 공백을 1칸 주면 좋을 것 같아요.
```suggestion
const likeRouter = require("./likeRouter");

router.use("/users", userRouter.router);
```